internal/controller/capstone_group/capstone_group.controller.go

```diff
@@ -323,3 +324,49 @@ func (cgc *CapstoneGroupController) GetListStudentHaveCapstoneGroup(ctx *gin.Con
 	}
 	response.SuccessResponse(ctx, http.StatusOK, output)
 }
+
```

Comment:
Please add this controller below the same path to easy review or check duplicate router